### PR TITLE
Corrupted Placement Group alert test case

### PR DESCRIPTION
**Pull request by fbalak**
- Add `test_corrupt_pg_alerts` test case.
- Add `create_dummy_osd` function. This can be used for Ceph storage manipulation as by using osd created by this function there is no lock on osd data.

### CODE REVIEW

**Comment by fbalak** [comment]
Logs: https://paste.fedoraproject.org/paste/uBBiD0MOG0C816j6sRJcmQ

File: tests/manage/monitoring/conftest.py [hunk 1]
```diff
@@ -342,3 +347,150 @@ def stop_osd():
     oc.exec_oc_cmd(f"scale --replicas=1 deployment/{osd_to_stop}")
 
     return measured_op
+
+
+def create_dummy_osd(deployment):
```
Comment (ebenahar) [inline tests/manage/monitoring/conftest.py]: Since it's not a fixture, thinking it would be better to have this function in another file, like helpers.py

Comment (mbukatov) [reply tests/manage/monitoring/conftest.py]: No, please don't add functions into helpers file withing a test directory tree.

Comment (mbukatov) [reply tests/manage/monitoring/conftest.py]: You could add it into ocsci module.

**Comment by fbalak** [comment]
@ebenahar Yes, it's done on lines starting on https://github.com/red-hat-storage/ocs-ci/pull/939/files#diff-a650626eeae1cbc843005d30e3a6c7feR415. It is done after the workload that is tested is done.

**Comment by vasukulkarni** [comment]
Nice to see this test!

**Pull request merged**